### PR TITLE
Feature/mc 13945 delete custom rules

### PR DESCRIPTION
**Pull request by RGitB**
#### Changes made

- Added docs for delete operation for custom rules

#### Related PRs
- [MC-13945](https://github.com/cloudops/cloudmc-stackpath-plugin/pull/336)

### CODE REVIEW

File: source/includes/stackpath/_custom_rules.md [hunk 1]
```diff
@@ -154,4 +154,38 @@ Attributes | &nbsp;
 `conditions.operation`<br/>*string* | Operation of the condition, one of: `EQUAL`, `NOT_EQUAL`, `CONTAINS`, `NOT_CONTAINS`, `BETWEEN` or `NOT_BETWEEN`.
 `conditions.header`<br/>*string* | Value of the header. Only for condition operation of type `HEADER`.
 `conditions.value`<br/>*string* | Value for which the condition holds.
-`conditions.endValue`<br/>*string* | Second value of the condition. Only for condition operation of type `IP_RANGE`.
\ No newline at end of file
+`conditions.endValue`<br/>*string* | Second value of the condition. Only for condition operation of type `IP_RANGE`.
+
+
+<!-------------------- DELETE A SITE -------------------->
```
Comment (fyrnaga) [inline source/includes/stackpath/_custom_rules.md]: Please change this comment to mention WAF custom rule

**Review by fyrnaga — APPROVED** [review]
Please make the small correction but looks good to me

File: source/includes/stackpath/_custom_rules.md (hunk 1 above)
Comment (RGitB) [reply source/includes/stackpath/_custom_rules.md]: 3767e60

**Review by GuillaumeVialle — APPROVED** [review]
LGTM

**Pull request merged**